src/docfx/build/metadata/Metadata.cs

```diff
@@ -21,5 +28,101 @@ where item.Match(file.FilePath)
 
             return JsonUtility.Merge(config.GlobalMetadata, fileMetadata);
         }
+
+        public static JObject GenerateRawMetadata(Context context, Document file, TableOfContentsMap tocMap, string html)
```

Comment:
Put these metadatas to PageModel and convert them to legacy format in LegacyMarkdown.cs:

- WordCount
- Locale
- Toc